### DISCUSSION TITLE
Upper sequence length limit in ESM Search

### DISCUSSION THREAD

**Discussion by agiani99** [facebookresearch/esm#444] [ESM Metagenomic Atlas]
Hi all,
I am wondering how could it cost to bring the actual sequence search upper limit from 400 to 2500 (upper limit fo AF2).
I am asking this because in several RNA viral sequences, the primary L-Protein (direct RNA transcript) can be as high as 4-5K residues.
This is one of the most relevant problem we are facing in the field: no prediction possible so far.
I attached a sample virus L-Protein sequence example for Crimean-Congo hemorrhagic fever virus [CCHFV_L_all3945.fasta](https://github.com/facebookresearch/esm/files/10324143/CCHFV_L_all3945.txt) 
Best
Andrea

**Comment by tomsercu** [comment]
The colabfold notebook supports longer sequences up to 900 residues, but that is out of distribution wrt training data.
For even longer proteins: Is it possible to split it up into domains and predict on individual domains or pair of domains?
